src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessNextDataSetVersionMappingsFunctionTests.cs

```diff
@@ -115,7 +26,7 @@ public abstract class CreateMappingsTests(
 
         protected async Task CreateMappings(Guid instanceId)
         {
-            var function = GetRequiredService<ProcessNextDataSetVersionMappingsFunction>();
+            var function = GetRequiredService<ProcessNextDataSetVersionMappingsFunctions>();
```

Comment:
I think this file might need to be moved from `ProcessNextDataSetVersionMappingsFunctionTests.cs` to `ProcessNextDataSetVersionMappingsFunctionsTests.cs`. Similar comment for the name of the abstract test class.

Reply:
Right you are, thanks for spotting!